### PR TITLE
Davers doubletree scraper

### PR DESCRIPTION
**Pull request by zzany**
Navigate to scrapers, covid19-vaccine-tracker, read README. To be run on cloudwatch, runs perpetually and scrapes every +/- 5 minutes
Added attribution in FAQ
image

### CODE REVIEW

File: scrapers/covid19-vaccine-tracker/backends/curative.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            location=sites["name"],
+            address="%s, %s, %s %s" % (sites["street_address_1"], sites["city"], sites["state"], sites["postal_code"]),
+            count=win["slots_available"],
+            vaccine_type=None,
+            details=("public: %d/%d, " % (win["public_slots_available"], win["public_slots"])) + \
+               ("first responder: %d/%d, " % (win["first_responder_slots_available"], win["first_responder_slots"])) + \
+               ("symptomatic: %d/%d" % (win["symptomatic_slots_available"], win["symptomatic_slots"]))
+        )
+
+    def humanize_date(self, d):
```
Comment (johndeyrup) [inline scrapers/covid19-vaccine-tracker/backends/curative.py:42]: we are going to end up storing epoch timestamps in the db, but this is fine for now

File: scrapers/covid19-vaccine-tracker/backends/cvs.py [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+    
+    def build_stores_data(self, payload=None):
+        return {
+            "requestMetaData": {
+                "appName": "CVS_WEB",
+                "lineOfBusiness": "RETAIL",
+                "channelName": "WEB",
+                "deviceType": "DESKTOP",
+                "deviceToken": "7777",
+                "apiKey": "a2ff75c6-2da7-4299-929d-d670d827ab4a",
```
Comment (johndeyrup) [inline scrapers/covid19-vaccine-tracker/backends/cvs.py:47]: Is this ok to be public?

Comment (zzany) [reply scrapers/covid19-vaccine-tracker/backends/cvs.py:47]: It's public on his repo, but we aren't using the CVS scraper anyway (doesn't work / get banned) so I'll cut it

**Comment by johndeyrup** [comment]
I think you need a file called lambda_function.py and a method
```
def lambda_handler(event, context):
	# my code
```

**Comment by johndeyrup** [comment]
The database is now set up and you can hit the apis so you can try populating the data into the postgresql database

**Pull request closed without merging**